docs/static/style-guide/formats.md

```diff
@@ -0,0 +1,203 @@
+> This library provides various functions for formatting the data in the preferred style.
+
+## Formatting Functions
+
+#### `sszvis.formatText(x)`
+
+Any given data is returned as a string and will be formatted as Text.
+
+```table
+rows:
+  - Function: formatText
+    Example: sszvis.formatText(16000)
+    Result: "'16000'"
```

Comment:
Used this to show that the returned format is a string. Not sure if I formatted this correctly.